components/base/sensorcontrolled/movestraight.go

```diff
@@ -22,35 +22,37 @@ const (
 func (sb *sensorBase) MoveStraight(
```

Comment:
[q]should we add a comment for each function describing how it uses controls?

Reply:
yeah good idea